### ISSUE TITLE
"init is not a known dfm command"

### ISSUE THREAD

**Issue by ynavott** [chasinglogic/dfm#49]
```
[ynavott-gram@ynavott-gram-ubuntu: /usr/local/bin]$ ./dfm
Usage:
    dfm [options] <command> [<args>...]
    dfm help
    dfm sync
    dfm link <profile>
[ynavott-gram@ynavott-gram-ubuntu: /usr/local/bin]$ ./dfm init profile-main
init is not a known dfm command.
``` This is a fresh clone which installed totally fine (at least it seems like it). How come init doesn't exist? Have tried using `dfm -v init profie-main`, `dfm -v i profile-main`, `dfm i profile-main` and it also doesn't work.

**Comment by chasinglogic** [comment]
Looks like somehow that command's code has gone missing. I'm fixing it now.

**Comment by chasinglogic** [comment]
Fixed and pushed 7.4.1

I'm working on integration tests that will catch these issues automatically in the future. Thanks for the bug report.
